blocks/checkbox/__label/checkbox__label.css

```
@@ -0,0 +1,43 @@
+.checkbox__label{
+    font-size: 18px;
+    line-height: 30px;
+    font-family: OpenSans, arial, sans-serif;
+    cursor: pointer;
+    position: relative;
+}
+
+.checkbox__label::before{
+    content:"";
+    position: absolute;
+    top: 50%;
+    left: -59px;
+    transform: translateY(-50%);
+    width: 35px;
+    height: 35px;
+    border-radius: 1px;
+    border: 2px solid rgb(229, 229, 229);
+    background-color: rgb(255, 255, 255);
+    transition: all .1s ease-in;
+}
+
+.checkbox__label::after{
+    content:url(../../../images/checkbox-tick.svg);
+    justify-content: center;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    left:-50.5px;
+    opacity: 0;
+}
+
+.checkbox__input:checked + .checkbox__label::after{
+    opacity: 1;
+}
+
+.checkbox__input:hover + .checkbox__label::before{
+    border-color: rgb(40, 54, 69);
+}
+
+.checkbox__input:active + .checkbox__label::before, .checkbox__input:focus + .checkbox__label::before{
```

```suggestion
.checkbox__input:active + .checkbox__label::before,
.checkbox__input:focus + .checkbox__label::before{
```
@NazarPauk Let's write each of those two selectors on a new line to make it easier for other developers to navigate our code 🙏